### ISSUE TITLE
Automatically bring back owner repos

### ISSUE THREAD

**Issue by Layla-P** [Layla-P/HacktoberfestProject#6]
When creating a new PR record when the "owner" is entered into the form, make an API call to bring back a drop down list of all the repos they own to select room.
This also validates the owner and repo to be real.
Could even bring back all the PRs on the repo, compare against user's name etc. 
Could be done with JavaScript.

(reach goal)

**Comment by CrypticEngima** [comment]
Added a GihubService to pull repo and pull requets from the github api #13

**Comment by Layla-P** [comment]
That will mean a request to and from the server. I think this may work better if it was client-side.
With a GitHub service we'll need to create an API Controller for the front end to call to bring back the list of repos/PRs etc.

**Comment by napalm684** [comment]
Are we still wanting to do something JS side. I could probably work on a typeahed for the owner field if that's what you want (sounds like it). Let me know.

**Comment by napalm684** [comment]
Scratch that I think someone beat me to it. Looking for the next item up for bids :-)

**Comment by CrypticEngima** [comment]
@napalm684  The current solution is just a temp usage from the URL via JS we have not implamented the Github lookups so this task is still available

**Comment by napalm684** [comment]
@CrypticEngima did I get the description correct then above of what we are after? I might have time to look at this today so I want to be clear what the goal is.

**Comment by CrypticEngima** [comment]
@napalm684  yes the goal is to use the Github API to bring back 1) owner name after typing a few characters to do a dropdown select. 2) the roepos for that owner once owner is selected  3) list of Prs once the Owner and Repo is selected.
